test/runtests.jl

```diff
@@ -91,7 +93,9 @@ try
     NEW = GZip.GZLIB_VERSION > "1.2.3.9"
     pos = position(gzfile)
     NEW && (pos2 = position(gzfile,true))
-    @test_throws ErrorException seek(gzfile, 100)   # can't seek backwards on write
+    try 
+        @test_throws ErrorException seek(gzfile, 100)   # can't seek backwards on write
```

Comment:
That is right, I did not notice that at first. But we see in other places that there are differences in the library.